pom.xml

```diff
@@ -397,6 +394,9 @@
 
 		<profile>
 			<id>jacoco</id>
+			<properties>
+				<opennlp.forkCount>1</opennlp.forkCount>
```

Comment:
u can change that to 2 or 3, when it was set to 1.0C it was utilizing all of the available cores and hence Travis was killing the job.

Setting it to 1 is meaningless, so i would change that to 2 or 3.